src/tools/swix/swcore/CompilerBackend/Nuget/NugetBackendCompiler.cs

```diff
@@ -53,7 +53,10 @@ public override void Generate(IEnumerable<Intermediate> intermediates, string ou
                 return;
             }
 
-            FileTransfer packageTransfer = FileTransfer.Create(null, Path.GetTempFileName(), outputPath, "NugetPackage", true);
+            int lastIndex = outputPath.LastIndexOf('.');
```

Comment:
So we're purposely deviating from the behavior of `nuget.exe pack`?  Probably should decline the bug then.